requirements.txt

```diff
@@ -2,3 +2,4 @@ django<2.0
 django-distill
 IGitt
 requests
+python-dateutil
```

Comment:
this is already merged

Reply:
What do you mean? The repo [requirements.txt](https://github.com/coala/community/blob/master/requirements.txt) doesn't contain it.